samples/nrf9160/coap_client/README.rst

```diff
@@ -58,3 +93,8 @@ In addition, it uses the following samples:
 
 From |NCS|
   * :ref:`secure_partition_manager`
+
+References
+**********
+
+`AT Commands Reference Guide`_
```

Comment:
I would leave this out - the AT commands aren't really relevant for this sample.
But you could add a link to the RFC (the full doc, not the link to the error code section)?

Reply:
Replaced this with a link to RFC 7252 (https://tools.ietf.org/html/rfc7252).